### PR TITLE
85787 Update confirmation page - form 10-10d

### PR DESCRIPTION
**Pull request by michaelclement**
## Are you removing, renaming or moving a folder in this PR?
- [X] No, I'm not changing any folders (skip to Summary and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

## Summary

This PR updates the confirmation page copy of form 10-10d.

- Hides the "Get Help" footer only on the confirmation pages for CHAMPVA forms (information is not accurate for this subset of forms on the confirmation page, but is useful during normal form flow)
- Updated "what happens next" copy
- Updated mailing/faxing copy
- Team: IVC CHAMPVA
- Flipper: in use

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/85787

## Testing done

- Manual
- Verified unit tests run and pass

## Screenshots

|         | Before | After |
| ------- | ------ | ----- |
| Desktop |Screenshot 2024-07-22 at 12 10 35|Screenshot 2024-07-22 at 12 24 10|

## What areas of the site does it impact?

IVC form 10-10d (Confirmation page, Get Help footer), IVC form 10-7959c (Get Help footer)

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [X] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [X] Browser console contains no warnings or errors.
- [X] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

NA

### CODE REVIEW

**Review by reiting — DISMISSED** [review]

**Review by balexandr — DISMISSED** [review]

**Review by cloudmagic80 — DISMISSED** [review]

**Review by cloudmagic80 — DISMISSED** [review]

**Review by balexandr — DISMISSED** [review]

File: src/applications/ivc-champva/10-10D/containers/ConfirmationPage.jsx [hunk 1]
```diff
@@ -111,13 +96,38 @@ export function ConfirmationPage(props) {
...
+        received by CHAMPVA.
+        <br />
+        <br />
+        If we have any questions, need additional information, or encounter any
+        issues, we will contact you.
+      </p>
+      <h2>How to contact us about your CHAMPVA application</h2>
+      <p>
+        If you have any questions about your application you can call the
+        CHAMPVA call center at 800-733-8387. We’re here Monday through Friday,
```
Comment (micahchiang) [inline src/applications/ivc-champva/10-10D/containers/ConfirmationPage.jsx:120]: I think it's relatively common for phone numbers to be clickable. Should this be a [va-telephone](https://design.va.gov/storybook/?path=/docs/components-va-telephone--docs) component?

File: src/applications/ivc-champva/10-10D/containers/ConfirmationPage.jsx [hunk 2]
```diff
@@ -111,13 +96,38 @@ export function ConfirmationPage(props) {
...
+        <br />
+        <br />
+        You can also contact us online through our Ask VA tool.
+        <br />
+        <br />
+        <VaLink text="Go to Ask VA" href="https://ask.va.gov/" />
+      </p>
+      <br />
+      <br />
       <a className="vads-c-action-link--green" href="https://www.va.gov/">
```
Comment (micahchiang) [inline src/applications/ivc-champva/10-10D/containers/ConfirmationPage.jsx:131]: I know it isn't part of this PR, but you could make this a [va-link-action](https://design.va.gov/storybook/?path=/docs/components-va-link-action--docs) component if you wanted to :smile:

**Review by micahchiang — DISMISSED** [review]
Couple comments about web component usage, but not a requirement.

File: src/applications/ivc-champva/10-10D/containers/ConfirmationPage.jsx (hunk 2 above)
Comment (michaelclement) [reply src/applications/ivc-champva/10-10D/containers/ConfirmationPage.jsx]: Updated in [this commit](https://github.com/department-of-veterans-affairs/vets-website/commit/884ec7575eae4b79a34fb9e6c0eed8fe10a3f2d8) - thanks!

File: src/applications/ivc-champva/10-10D/containers/ConfirmationPage.jsx (hunk 1 above)
Comment (michaelclement) [reply src/applications/ivc-champva/10-10D/containers/ConfirmationPage.jsx]: Updated in [this commit](https://github.com/department-of-veterans-affairs/vets-website/commit/884ec7575eae4b79a34fb9e6c0eed8fe10a3f2d8) - thanks!

**Review by micahchiang — APPROVED** [review]

**Pull request merged**